app/src/main/java/org/oppia/app/utility/SplitScreenManagerModule.kt

```diff
@@ -0,0 +1,16 @@
+package org.oppia.app.utility
+
+import android.content.Context
+import dagger.Module
+import dagger.Provides
+import org.oppia.app.application.ApplicationContext
+
+/** A module for [SplitScreenManager] for dagger injection*/
```

Comment:
Nit: `/** A module for [SplitScreenManager] for dagger injection. */`

Reply:
Done